Exit code: 1
Stdout:

Stderr:
Traceback (most recent call last):
  File "/Users/valeriaenriquezlimon/Documents/meta-h-generator/llm-metaheuristics/outputs-results/ollama_output_Rastrigin_15_20250111_002353/execution_iteration_5.py", line 38, in <module>
    print('x_best = {}, f_best = {}'.format(*met.get_solution()))
                                             ^^^^^^^^^^^^^^^^^^
  File "/Users/valeriaenriquezlimon/Documents/meta-h-generator/llm-metaheuristics/metaheuristic.py", line 173, in get_solution
    return self.historical['position'][-1], self.historical['fitness'][-1]
           ~~~~~~~~~~~~~~~^^^^^^^^^^^^
KeyError: 'position'
